test/Microsoft.AspNet.Mvc.Core.Test/ActionResults/RedirectToActionResultTest.cs

```diff
@@ -59,7 +60,7 @@ public void RedirectToAction_Execute_ThrowsOnNullUrl()
             };
 
             // Act & Assert
-            ExceptionAssert.ThrowsAsync<InvalidOperationException>(
+            await ExceptionAssert.ThrowsAsync<InvalidOperationException>(
```

Comment:
The change here is because I was receiving a Roslyn compilation exception mentioning that this method could return even before this async call is finished(which is true)...but not sure why this exception only surfaces with my change...